service/src/main/java/uk/gov/hmcts/reform/fpl/service/docmosis/CaseSubmissionGenerationService.java

```diff
@@ -121,10 +112,8 @@ public DocmosisCaseSubmission getTemplateData(final CaseData caseData) {
             .groundsThresholdReason(caseData.getGrounds() != null
                 ? buildGroundsThresholdReason(caseData.getGrounds().getThresholdReason()) : DEFAULT_STRING)
             .thresholdDetails(getThresholdDetails(caseData.getGrounds()))
-            .annexDocuments(
-                featureToggleService.isApplicationDocumentsEventEnabled()
-                    ? annexGenerator.generate(caseData) : buildDocmosisAnnexDocuments(caseData)
-            ).userFullName(idamClient.getUserInfo(requestData.authorisation()).getName());
+            .annexDocuments(annexGenerator.generate(caseData))
```

Comment:
Theres a switch in the document template also. DocmosisAnnexDocuments.builder().featureToggleOn(true) that can be removed. Since the feature is toggled in prod, if we upload the template FL-PLW-APP-ENG-00095.doc with the amended template (without the switches first, then we can remove the code also).

Reply:
Good spot, did not realise FT was in templates also.